.github/workflows/test-fork-prs.yml

```diff
@@ -204,6 +204,10 @@ jobs:
   e2e:
     uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@main
     needs: [setup, setup-cache]
+    permissions:
+      pull-requests: write # used to remove label
+      id-token: write # This is required for requesting the JWT
+      contents: read # This is required for actions/checkout
```

Comment:
These permissions were required to use IAM roles with OIDC. This was originally there before, but got missed out during the refactor.